flexget/plugins/urlrewrite_divxatope.py

```diff
@@ -8,24 +8,59 @@
 from flexget.utils import requests
 from flexget.utils.soup import get_soup
 
+from flexget.entry import Entry
+from flexget.utils.search import torrent_availability, normalize_unicode
+
 log = logging.getLogger('divxatope')
 
+CATEGORIES = {
```

Comment:
I will remove the categories. But one question, in the scheme what are the needed changes?